### PR TITLE
fixed a small typo in cod3.c

### PR DESCRIPTION
**Pull request by dunkyp**
currently can't build with BUILD=debug

### CODE REVIEW

File: src/ddmd/backend/cod3.c [hunk 1]
```diff
@@ -2479,7 +2479,7 @@ void genmovreg(CodeBuilder& cdb,unsigned to,unsigned from)
 {
 #if DEBUG
         if (to > ES || from > ES)
-                printf("genmovreg(c = %p, to = %d, from = %d)\n",c,to,from);
+                printf("genmovreg(c = %p, to = %d, from = %d)\n",cdb,to,from);
```
Comment (rainers) [inline src/ddmd/backend/cod3.c]: I guess it should be `&cdb`. Please change the format string, too.

**Comment by wilzbach** [comment]
> currently can't build with BUILD=debug

We should definitely add this to the CI setup. Circle CI maybe?
CC @CyberShadow

File: src/ddmd/backend/cod3.c (hunk 1 above)
Comment (WalterBright) [reply src/ddmd/backend/cod3.c]: I just changed it to:
```
                printf("genmovreg(to = %d, from = %d)\n",to,from);
```

**Review by rainers — APPROVED** [review]
edited online.

**Pull request merged**